hw/mcu/nordic/nrf52xxx/nrf52.ld

```diff
@@ -145,6 +145,10 @@ SECTIONS
         *(.data*)
 
         *(.jcr)
+
+        __MYNEWT_SENSITIVE_START = .;
+        *(.mynewt_sensitive*)
+        __MYNEWT_SENSITIVE_END = .;
```

Comment:
So, I think these should be consistent with the other variables, so, can you please make it lowercase.

Reply:
lower cased